res/job_queue/queue.py

```diff
@@ -628,6 +630,8 @@ def add_ensemble_evaluator_information_to_jobs_file(
             with open(f"{q_node.run_path}/{JOBS_FILE}", "r+") as jobs_file:
                 data = json.load(jobs_file)
 
+                data["experiment_id"] = experiment_id
+                data["experiment_url"] = experiment_url
```

Comment:
needs to be added on the condition that the feature flag for experiment server is set

also, no work was made on the experiment "ingest" url, so the reporter will likely fail